lib/proto/attach-link.js

```diff
@@ -56,6 +56,8 @@ var linkAction = function(el, event) {
 module.exports = function(el) {
   var that = this
 
+  el.setAttribute("data-pjax-enabled", "true");
```

Comment:
I just saw that I am already using some attr in  pjax/lib/proto/attach-link.js. Maybe we can just check that `data-pjax-click-state` (attrClick) exist (instead of creating a new one) ?